### PR TITLE
Feature/account postgres sink

### PR DESCRIPTION
**Pull request by bryzettler**

### CODE REVIEW

File: packages/account-postgres-sink-service/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+RUN yarn install
+
+COPY src src
+COPY tsconfig.build.json tsconfig.json
+
+RUN yarn global add typescript
+RUN yarn run build
+
+RUN npm prun --production
```
Comment (ChewingGlass) [inline packages/account-postgres-sink-service/Dockerfile:16]: npm prun lol

File: packages/account-postgres-sink-service/package.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    "@helium/address": "^4.6.2",
+    "@helium/helium-sub-daos-sdk": "^0.0.46",
+    "@helium/idls": "^0.0.46",
+    "@helium/spl-utils": "^0.0.46",
+    "@helium/voter-stake-registry-sdk": "^0.0.46",
+    "@metaplex-foundation/js": "0.18.3",
+    "@metaplex-foundation/mpl-token-metadata": "^2.2.3",
+    "@solana/spl-token": "^0.3.6",
+    "@solana/web3.js": "^1.43.4",
+    "angry-purple-tiger": "^1.0.5",
```
Comment (ChewingGlass) [inline packages/account-postgres-sink-service/package.json:46]: Doesn't seem like it needs all these deps

**Review by ChewingGlass — CHANGES_REQUESTED** [review]

File: packages/account-postgres-sink-service/src/utils/santizeAccount.ts
Comment (ThornM9) [inline packages/account-postgres-sink-service/src/utils/santizeAccount.ts:1]: Typo in file name

**Review by ThornM9 — APPROVED** [review]

File: packages/account-postgres-sink-service/src/server.ts [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  try {
+    // HSD
+    await upsertProgramAccounts({
+      programId: HSD_PROGRAM_ID,
+      accounts: [
+        { type: "DaoV0", table: "daos", schema: "hsd" },
+        { type: "SubDaoV0", table: "sub_daos", schema: "hsd" },
+        {
+          type: "SubDaoEpochInfoV0",
+          table: "sub_dao_epoc_info",
```
[the remark this answers is not in the document]
Comment (ChewingGlass) [reply packages/account-postgres-sink-service/src/server.ts:67]: Can we just make this all a YAML config? And have one endpoint that causes a re-trigger on all tables. The endpoints helius will be hitting won't be structured like this anyway.

[the remark this answers is not in the document]
Comment (ChewingGlass) [reply packages/account-postgres-sink-service/src/server.ts:67]: I'd rather not have to re push docker containers to change this config

File: packages/account-postgres-sink-service/src/utils/upsertProgramAccounts.ts [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+              ...updateOnDuplicateFields,
+            ],
+          }
+        );
+
+        await model.destroy({
+          transaction: t,
+          where: {
+            refreshed_at: {
+              [Op.ne]: sequelize.fn("NOW"),
```
[the remark this answers is not in the document]
Comment (bryzettler) [reply packages/account-postgres-sink-service/src/utils/upsertProgramAccounts.ts:144]: Is it? they're both in the same transaction I assumed the would be the same now

**Review by ChewingGlass — APPROVED** [review]

**Pull request merged**